public/css/styles.css

```diff
@@ -0,0 +1,673 @@
+
+h1:after, h1:before {
+  top: 51%;
+  width: 32%;
+  content: "";
+  position: absolute;
+  overflow: hidden;
+  height: 1px;
+  background-color: #585050;
+  margin-left: 3%;
+}
+
+h1:before {
+  margin-left: -35%;
+  text-align: right;
+}
+
+header {
+  margin-top: 3em;
+  margin-bottom: -3em;
+}
+
+.section-orange > h1:before, .section-orange > h1:after {
+  background-color: #fff;
+}
+
+.section-black > h1:before, .section-black > h1:after {
+  background-color: #fff;
+}
+
+.clip {
+  margin-left: 3em;
+}
+
+.subheader {
+  position: relative;
+  overflow: hidden;
+  z-index: 1;
+  margin-bottom: 2em;
+  margin-top: 2em;
+}
+.subheader:before, .subheader:after {
+  top: 51%;
+  width: 20%;
+  content: "";
+  position: absolute;
+  overflow: hidden;
+  height: 1px;
+  background-color: #585050;
+  margin-left: 2%;
+}
+.subheader-skills:before, .subheader-skills:after {
+  background-color: #fff;
+}
+.subheader:before {
+  margin-left: -22%;
+  text-align: right;
+}
+.section-orange > .subheader:before, .section-orange > .subheader:after {
+  background-color: #fff;
+}
+.section-white > .subheader:before, .section-white > .subheader:after {
+  background-color: #000;
+}
+
+.column-two {
+  -webkit-column-count: 2;
+     -moz-column-count: 2;
+     -ms-column-count: 2;
+          column-count: 2;
+}
+.column-right {
+  float: right;
+}
+.section-orange {
+  background-color: #FF6124;
+  color: #fff;
+  padding-bottom: 2em;
+}
+
+.section-black {
+  background-color: #585050;
+  color: #fff;
+}
+
+/*---Navigation menu css---*/
+.hidden-class {
+  display: inline-block;
+}
+
+.nav-li {
+  margin-left: 3em;
+}
+
+.drop-down {
+  font-size: 1.2em;
+}
+
+/*---main header css---*/
+.header-section {
+  font-size: 2em;
+  font-family: helvetica;
+  height: 17em;
+}
+
+.header-section {
+  background-image: url("../imgs/top-bkgrnd.jpg");
+}
+
+/*---services section css---*/
+.flex-container {
+  width: 100%;
+  display: flex;
+  display: -webkit-flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  align-items: flex-start;
+  -webkit-justify-content: center;
+  -webkit-flex-direction: row;
+  -webkit-flex-wrap: wrap;
+}
+
+ .flex-item-1 {
+  width: 100%;
+  height: 24em;
+  text-align: center;
+}
+
+.flex-item-2 {
+  width: 100%;
+  height: 20em;
+  text-align: center;
+  margin-left: -.5em;
+}
+.flex-item-3 {
+  width: 22%;
+  height: 20em;
+  text-align: center;
+}
+.flex-item-4 {
+  width: 22.5%;
+  text-align: center;
+}
+
+.table-section {
+  width: 100%;
+  display: flex;
+  display: -webkit-flex;
+  justify-content: center;
+  padding-bottom: 3em;
+  padding-right: 5%;
+  padding-left: 5%;
+  -webkit-justify-content: center;
+}
+
+table {
+  position: relative;
+  float: left;
+  margin-right: 1em;
+  font-size: .8em;
+  margin-bottom: 2em;
+}
+th {
+  background-color: #703A20;
+  color: #fff;
+}
+tr {
+  background-color: #fff;
+  color: #000;
+}
+tr:not(.table-head) {
+  border-bottom: .1em solid #FF6124;
```

Comment:
I'm not seeing this actually being applied. But in any case, using `:not` should be a last resort; if you find yourself needing it, take another look at how you're doing things. Perhaps this could go on the `th`? Especially if you removed some of the unnecessary table structure (see my notes on the markup.)